packages/keystone/rateLimiting/config.utils.js

```diff
@@ -0,0 +1,32 @@
+const Ajv = require('ajv')
+
+const ajv = new Ajv()
+
+const PLUGIN_OPTIONS_SCHEMA = {
+    type: 'object',
+    properties: {
+        customQuotas: { type: 'object', additionalProperties: { type: 'number' } },
+        queryWeight: { type: 'number' },
+        mutationWeight: { type: 'number' },
+        window: { type: 'string' },
```

Comment:
What is the goal of this field?

Reply:
Sliding window of quotas (TTL)

Reply:
Maybe then we should call the field `quotaWindow`?
As your const `DEFAULT_QUOTA_WINDOW`